### ISSUE TITLE
cerulean: body text is too light, and is the same color as text-muted

### ISSUE THREAD

**Issue by Kwpolska** [thomaspark/bootswatch#825]
The default text color, `#868e96`, is too light for body text according to [WebAIM Color Contrast Checker](https://webaim.org/resources/contrastchecker/). Moreover, `.text-muted` uses the same color, which means this style option does not work.
![image](https://user-images.githubusercontent.com/327323/39084630-f01d8b00-4578-11e8-9267-16a6472690b0.png)
